sqlair.go

```diff
@@ -169,6 +178,7 @@ func (q *Query) Get(outputArgs ...any) error {
 }
 
 // Iter returns an Iterator to iterate through the results row by row.
+// Close must be run once iteration is finished.
```

Comment:
```suggestion
// Iterator.Close must be run once iteration is finished.
```
The same comment about godoc and picking up names above applies here. Whatever Intellij highlights should work.